icu4c/source/test/intltest/numbertest_api.cpp

```diff
@@ -3433,6 +3433,15 @@ void NumberFormatterApiTest::roundingOther() {
             u"0.0",
             u"0.0");
 
+    assertFormatSingle(
+            u"Large integer increment",
+            u"precision-increment/24000000000000000000000",
+            u"precision-increment/24000000000000000000000",
+            NumberFormatter::with().precision(Precision::incrementExact(24, 21)),
+            Locale::getEnglish(),
+            3.1e22,
+            u"24,000,000,000,000,000,000,000");
+
```

Comment:
could you add several more test cases with 1, 5, 10, 25 as increment and ECMA402 limit for maginitute

Reply:
fwiw, this feature is already quite well tested and it took work to make all the existing tests pass, but I'll add some more to align with ECMA-402 as you suggested.

Reply:
Added more tests. Also to Java.